go-mod-tidy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Root of repo
+ROOT_DIR=${1:-'..'}
+
+GOMODULE_PATHS=("core/network/fabric-interop-cc/libs/utils"
+"core/network/fabric-interop-cc/libs/assetexchange"
+"core/network/fabric-interop-cc/interfaces/asset-mgmt"
+"core/network/fabric-interop-cc/contracts/interop"
+"sdks/fabric/go-sdk"
+"samples/fabric/go-cli"
+"samples/fabric/simpleasset"
+"samples/fabric/satpsimpleasset"
```

Comment:
Remove this. The `satp` code is not part of Weaver.

Reply:
Done.